lib/galaxy/jobs/__init__.py

```diff
@@ -330,6 +313,12 @@ def __parse_job_conf_xml(self, tree):
 
         log.debug('Done loading job configuration')
 
+    def _parse_handler(self, handler_element):
+        for plugin in handler_element.findall('plugin'):
+            if id not in self.handler_runner_plugins:
```

Comment:
😱 So we would seem to have no test coverage in any of our tests for that. Thanks for the catch 🦅👀!